app/views/documents/index.html.erb

```diff
@@ -56,17 +56,17 @@
           <p class="govuk-body govuk-!-margin-bottom-1">
             Date received: <%= document.created_at.strftime("%e %B %Y") %>
           </p>
-          <% if document.numbers.present? %>
+          <% unless document.numbers.blank? %>
```

Comment:
Smallest of quibbles, but this change feels like making it into a double negative? Really don't mind though.